src/jit/codegenarm64.cpp

```diff
@@ -275,11 +275,6 @@ void CodeGen::genPrologSaveReg(regNumber reg1,
 
     if (spDelta != 0)
     {
-        // If saving a single callee-save register, and we need to change SP, the offset cannot be zero. It must be 8 to account
-        // for alignment.
-        assert(spOffset != 0);
-        assert(spOffset == REGSIZE_BYTES);
-
         // generate sub SP,SP,imm
```

Comment:
Can you change the function comment to correctly name the function as genPrologSaveReg (it currently starts "genPrologSaveRegPair: ")